src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -256,15 +257,18 @@ s_no_extra_traits! {
         pub f_ffree: ::c_long,
         pub f_fsid: ::fsid_t,
         pub f_owner: ::uid_t,
-        pub f_type: i32,
-        pub f_flags: i32,
+        pub f_type: ::c_int,
+        pub f_flags: ::c_int,
         pub f_syncwrites: ::c_long,
         pub f_asyncwrites: ::c_long,
         pub f_fstypename: [::c_char; 16],
-        pub f_mntonname: [::c_char; 90],
+        pub f_mntonname: [::c_char; 80],
```

Comment:
Do we have constants that we could use here?

Reply:
The C source defines `MFSNAMELEN` and `MNAMELEN`, but I don't see any equivalents in rust's libc. I can add these as constants if you'd prefer.

Reply:
You're right, none of the other targets use a constant. I guess it's fine as it is.